test/kubemark/resources/start-kubemark-master.sh

```diff
@@ -424,6 +585,25 @@ function start-kubemaster-component() {
 	else
 		local -r component_docker_tag=$(cat ${KUBE_BINDIR}/${component}.docker_tag)
 		sed -i -e "s@{{${component}_docker_tag}}@${component_docker_tag}@g" "${src_file}"
+		if [ "${component}" == "kube-apiserver" ]; then
+			local audit_policy_config_mount=""
+			local audit_policy_config_volume=""
+			if [[ "${ENABLE_APISERVER_ADVANCED_AUDIT:-}" == "true" ]]; then
+				read -d '' audit_policy_config_mount << EOF
+    - name: auditpolicyconfigmount
+      mountPath: ${audit_policy_file}
+      readOnly: true
+EOF
+				read -d '' audit_policy_config_volume << EOF
+  - name: auditpolicyconfigmount
```

Comment:
I think moving it 4 spaces left && moving a variable in the yaml file to the beginning of the line would be more readable

Reply:
Done.. thanks.